pydis_site/apps/content/resources/guides/pydis-guides/contributing/obtaining-discord-ids.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Obtaining Discord IDs
+description: How to obtain Discord IDs to set up the bots.
+icon: fab fa-discord
+---
+First, enable developer mode in your client so you can easily copy IDs.
+
+1. Go to your `User Settings` and click on the `Appearance` tab.
+2. Under `Advanced`, enable `Developer Mode`.
+
+#### Guild ID
+
+Right click the server icon and click `Copy ID`.
+
+#### Channel ID
+
+Right click a channel name and click `Copy ID`.
+
+#### Role ID
+
+Right click a role and click `Copy ID`.
+The easiest way to do this is by going to the role list in the guild's settings.
```

Comment:
Perhaps include instructions on how to get to the role list?

Reply:
Hmm it's literally just entering the server's settings and then you have a roles tab. Not sure what information to add